packages/safe-core-sdk/src/utils/transactions/utils.ts

```diff
@@ -37,16 +38,22 @@ export async function standardizeSafeTransactionData(
     refundReceiver: tx.refundReceiver || ZERO_ADDRESS,
     nonce: tx.nonce ?? (await safeContract.getNonce())
   }
-  const safeTxGas =
-    tx.safeTxGas ??
-    (await estimateTxGas(
-      safeContract,
-      ethAdapter,
-      standardizedTxs.to,
-      standardizedTxs.value,
-      standardizedTxs.data,
-      standardizedTxs.operation
-    ))
+  let safeTxGas: number
+  const safeVersion = await safeContract.getVersion()
+  if (semverSatisfies(safeVersion, '>=1.3.0')) {
```

Comment:
Maybe put `'>=1.3.0'` in some sort of config, [similar to safe-react](https://github.com/gnosis/safe-react/blob/a2cd631b8f765c10a1fbf55de9290121e5da0e9b/src/logic/safe/utils/safeVersion.ts)?